images/cfw/opt/x1plus/lib/python/x1plus/services/x1plusd/__init__.py

```diff
@@ -18,6 +19,9 @@ async def main():
     settings = SettingsService(router=router)
     ota = OTAService(router=router, settings=settings)
     ssh = SSHService(settings=settings)
+    if settings.get("polar_cloud", False):
+        polar_cloud = PolarPrintService(settings=settings)
+        asyncio.create_task(polar_cloud.begin())
```

Comment:
This is fine, but this means that if `polar_cloud` is turned off, then you will have to reboot the printer in order to get it to load `PolarPrintService` (or otherwise restart `x1plusd`.  For now, I am OK with this, out of an abundance of caution, but probably later we will want `PolarPrintService` to be loaded all the time and then just be careful about enabling or disabling itself.

Reply:
Although actually, one more thought: if we *are* being careful about what we load or do not load here, maybe the `import` should be in this `if` block?  That way we can avoid the startup cost of loading all of `engineio` and its dependencies.  I'm not sure how much this actually costs in reality.

Reply:
Yes, short term, my plan was to have to reboot each time. In fact, I think you'd have to rebuild the entire OS and ssh it across, wouldn't you? I _think_ @jwise we talked about this, but I _know_ I was doing it to avoid bloat, which is honestly something that matters to me.